docs_2.0/manual_EN/4.ngql-guide/16.query-tuning-statements/1.explain-and-profile.md

```diff
@@ -0,0 +1,112 @@
+# EXPLAIN and PROFILE
+
+`EXPLAIN` helps output the execution plan of an nGQL statement without executing the statement, while `PROFILE` executes the statement, then output the execution plan as well as the execution profile. You can optimize the queries for better performance with the execution plan and profile.
```

Comment:
splitting the first sentence into two sentences is suggested, which could exert less stress to readers.